modules/engage-theodul-plugin-tab-downloads/package.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "engage-theodul-plugin-controls",
+  "private": true,
+  "scripts": {
+    "eslint": "eslint --config ../../docs/checkstyle/eslintrc.js --rule '{\"no-unused-vars\": \"off\"}' src/main/resources/**/*.js"
+  },
+  "devDependencies": {
+    "eslint": "^6.6.0",
```

Let's use a recent ESLint version from the start:

```suggestion
    "eslint": "^7.6.0",
```